packages/lit-dev-content/site/tutorials/content/async-directive/03.md

```diff
@@ -40,30 +40,18 @@ Last, update the render method to use `format`:
 You should now see the string updated to say "just now". But notice that it
 doesn't yet update on its own.
 
-<aside class="info">
+{% aside  "info" %}
 
-Note that standard `Directive`s like what we created with here have a lot more
-powers than we're using so far: they are stateful between renders and can access
-the underlying DOM they are associated with. See [Custom
-directives](/docs/templates/custom-directives/#creating-class-based-directives)
-for more details.
-
-<br>
+Directives can keep state, operate outside of Lit's lifecycle, and safely access
```

Comment:
I feel like this one needs updates, but I'm not sure these edits retain enough of the original. Can we just turn this into a no-header aside with the original text for now?